README.md

```diff
@@ -59,3 +59,12 @@ Another possible way is to create a different property file like application-cus
 
 The application has a built-in bug that you can trigger by
 navigating to the path `/is-it-coffee-time`.
+
+## Agent mode (elasticapm/opentelemetry)
+
+The Opbeans Java can use the APM Agent Java or the OpenTelemetry java implementation
+in order to choose one of other you have to set the environment variable APM_AGENT_TYPE
+to `elasticapm` when you use the APM Agent Java and `opentelemetry` to use OpenTelemetry java implementation.
```

Comment:
Now there is also the `none` value that will make Opbeans run without any instrumentation agent.

In this case it means that if OTel environment variables are properly et the OTel SDK will allow the application to send spans created through manual instrumentation.